lpd/tests/test_qualitative_data_analysis.py

```diff
@@ -51,3 +54,49 @@ def test_make_document(self, answers, expected_document):
         and performs stemming for inflected forms of 'teach' root.
         """
         self.assertEqual(qda.make_document(answers), expected_document)
+
+    @data(
+        (
+            [
+                'Teaching is awesome, I love it.',
+                'I have many teachers in my family, I thought of becoming one.',
+                'I would love to remove gaps among marginalized social groups.',
+                (
+                    'I need to improve my knowledge and skill, having worked in different places in the world,'
+                    'I\'ve gathered better understanding of different cultures.'
+                )
+            ],
+            {
+                'kc_id_1_set-me': 0.03020547,
+                'kc_id_2_set-me': 0.0302166,
+                'kc_id_3_set-me': 0.03020543,
+                'kc_id_4_set-me': 0.03020937,
+                'kc_id_5_set-me': 0.03021411,
+                'kc_id_6_set-me': 0.81875085,
+                'kc_id_7_set-me': 0.03019817,
```

Comment:
@tomaszgy The `_set-me` suffix is kind of confusing in this context, I think we can remove it :)

Reply:
@itsjeyd Good catch :) Just removed the suffix.